addons/default/visiosoft/advs-module/src/Support/Command/CheckModuleInstalled.php

```diff
@@ -0,0 +1,23 @@
+<?php namespace Visiosoft\AdvsModule\Support\Command;
+
+use Anomaly\Streams\Platform\Addon\Module\Contract\ModuleRepositoryInterface;
+
+class CheckModuleInstalled
+{
+    protected $moduleNamespace;
+    protected $checkEnabled;
+
+    public function __construct($moduleNamespace, $checkEnabled = true)
+    {
+        $this->moduleNamespace = $moduleNamespace;
+        $this->checkEnabled = $checkEnabled;
+    }
+
+    public function handle(ModuleRepositoryInterface $moduleRepository)
+    {
+        if ($module = $moduleRepository->findBy('namespace', $this->moduleNamespace)) {
```

Comment:
$installed_modules = app('module.collection')->installed();
        $installed_modules = $installed_modules->merge(app('extension.collection')->installed());

Reply:
such a function is available.

Reply:
where is this function I cant see it, if there is a function like this then let me know where it is

Reply:
You can view the stream platform.

Reply:
function name?

Reply:
söylemek istediğimiz yeni bir query yazmana ihtiyaç yoktur.
Bunun stream platformda kullanılabileceğin bir fonksiyon mevcuttur.

https://github.com/openclassify/openclassify/pull/713#discussion_r502415686

view addBlock extension

Reply:
it's already merged